docs/automatic-redirects.rst

```diff
@@ -20,10 +20,10 @@ The reasoning behind this is that RTD organizes the URLs for docs so that multip
 
 Among all the multiple versions of docs, you can choose which is the "default" version for RTD to display, which usually corresponds to the git branch of the most recent official release from your project.
 
-rtfd.org
+readthedocs.org
```

Comment:
This shouldn't be updated

Reply:
What are PO files here?

Reply:
These https://github.com/readthedocs/readthedocs.org/pull/5871/files#diff-416b5b299618cc307035d37fc0899f2e

And, not, those should live in repo, but not modified in this PR

Reply:
@stsewd I've made the changes, can you check now?